src/R/send-charts.R

```diff
@@ -0,0 +1,203 @@
+library(dplyr)
+library(ggplot2)
+library(reshape2)
+library(stringr)
+
+### Variables for all charts ###
+
+df_historical <- read.csv("../../target/historic-data.csv")
+remove_colons <-  function(x) {str_replace(x, ':', '')}
+df_historical <- mutate_all(df_historical, funs(remove_colons))
+df_historical <- df_historical %>%
+  filter(need.1 != "NONSEND")
+
+n_hist_years <-  df_historical %>%
+  summarise(n_distinct(calendar.year))
+
+hist_proj_split <- n_hist_years[1,] + 1.5
+
+
+###  Send by academic year ###
+
+df_projected_ay <- read.csv("../../target/output-ay.csv")
+df_projected_ay <- df_projected_ay[,c("calendar.year","academic.year","mean")]
+           
+df_historical_ay <- df_historical %>%
+  group_by(academic.year.1, calendar.year) %>%
+  count()
+
+names(df_historical_ay)[1] <- 'academic.year'
+names(df_historical_ay)[3] <- 'mean'
+
+df_ay <- rbind(df_historical_ay[,c(2,1,3)], df_projected_ay)
+
+df_ay <- transform(df_ay, academic.year = as.numeric(academic.year))
```

Comment:
It's because `academic.year` is a character type when it's loaded from the csv....was causing me much pain trying to implement lines 38 to 42 until I realised this.